<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="33e4e0f6-f61c-421f-b330-100e892d75ad" name="jhaveard@CAPEHORN 2010-02-03 21:58:50" runUser="capehorn\jhaveard" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2006">
  <TestRunConfiguration name="Local Test Run" id="f87716d9-9b5a-43ed-b471-b2677316cf16">
    <Description>This is a default test run configuration for a local test run.</Description>
    <Deployment runDeploymentRoot="C:\_CODE_DEPLOYED\_CODE_NO_SERVER\InspireServer\Events Server\_RebuildAll\TestResults\jhaveard_CAPEHORN 2010-02-03 21_58_50">
      <DeploymentItem filename="C:\_CODE_DEPLOYED\_CODE_NO_SERVER\InspireServer\Events Server\Inspire.Server.DirectionManager\Direction\Source\Service Interface\Direction.ServiceImplementation\bin\Debug\Inspire.Server.Direction.ServiceImplementation.dll" />
      <DeploymentItem filename="C:\_CODE_DEPLOYED\_CODE_NO_SERVER\InspireServer\Events Server\Inspire.Server.Events\Inspire.Events\Source\Resource Access\Inspire.Feeds.DataAccess\bin\Debug\Inspire.Server.Events.DataAccess.dll" />
    </Deployment>
    <TestTypeSpecific />
  </TestRunConfiguration>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <Times creation="2010-02-03T21:58:50.2679169-05:00" queuing="2010-02-03T21:58:58.0343611-05:00" start="2010-02-03T21:58:58.4253835-05:00" finish="2010-02-03T21:59:07.0788784-05:00" />
  <TestDefinitions>
    <UnitTest name="GetEventsUsingEndpoint" storage="c:\_code_deployed\_code_no_server\inspireserver\events server\_rebuildall\events.unit.tests\bin\debug\events.unit.tests.dll" id="b12fa771-bafe-5678-e3d7-5dc36105e86d">
      <Css projectStructure="" iteration="" />
      <Owners>
        <Owner name="" />
      </Owners>
      <Execution id="66e4fd05-fe73-4d61-8ae6-76dbeb42c425" />
      <TestMethod codeBase="C:\_CODE_DEPLOYED\_CODE_NO_SERVER\InspireServer\Events Server\_RebuildAll\Events.Unit.Tests\bin\Debug\Events.Unit.Tests.dll" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="Events.Unit.Tests.EventTests, Events.Unit.Tests" name="GetEventsUsingEndpoint" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="b12fa771-bafe-5678-e3d7-5dc36105e86d" executionId="66e4fd05-fe73-4d61-8ae6-76dbeb42c425" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="66e4fd05-fe73-4d61-8ae6-76dbeb42c425" testId="b12fa771-bafe-5678-e3d7-5dc36105e86d" testName="GetEventsUsingEndpoint" computerName="CAPEHORN" duration="00:00:06.6934265" startTime="2010-02-03T21:58:58.6773979-05:00" endTime="2010-02-03T21:59:06.8348645-05:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d">
      <Output>
        <ErrorInfo>
          <Message>Test method Events.Unit.Tests.EventTests.GetEventsUsingEndpoint threw exception:  System.ServiceModel.FaultException`1[Events.Unit.Tests.EventServiceReference.GeneralFaultContract]: Error retrieving HospitalityEvents from database. Could not find stored procedure 'events_get_with_event_definition_id_v1'..</Message>
          <StackTrace> 
Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc&amp; rpc)
   at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)
Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData&amp; msgData, Int32 type)
   at Events.Unit.Tests.EventServiceReference.EventServiceContract.GetEvents(GetEventsRequestMessage request)
   at Events.Unit.Tests.EventServiceReference.EventServiceContractClient.Events.Unit.Tests.EventServiceReference.EventServiceContract.GetEvents(GetEventsRequestMessage request) in c:\_code_deployed\_code_no_server\inspireserver\events server\_rebuildall\events.unit.tests\service references\eventservicereference\reference.cs:line 1660
   at Events.Unit.Tests.EventServiceReference.EventServiceContractClient.GetEvents(String EventDescriptionID) in c:\_code_deployed\_code_no_server\inspireserver\events server\_rebuildall\events.unit.tests\service references\eventservicereference\reference.cs:line 1666
   at Events.Unit.Tests.EventTests.GetEventsUsingEndpoint() in C:\_CODE_DEPLOYED\_CODE_NO_SERVER\InspireServer\Events Server\_RebuildAll\Events.Unit.Tests\EventTests.cs:line 94
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>